Fluid/Filters/NumberFilters.cs

```diff
@@ -19,6 +22,27 @@ public static FilterCollection WithNumberFilters(this FilterCollection filters)
             return filters;
         }
 
+        public static FluidValue Abs(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            return NumberValue.Create(Math.Abs(input.ToNumberValue()), true);
+        }
+
+        public static FluidValue AtLeast(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            var first = arguments.At(0);
+            double num = first.ToNumberValue();
+
+            return NumberValue.Create(Math.Max(input.ToNumberValue(), num), true);
```

Comment:
That's not what I meant. You shouldn't have to call `Create` at all. Or use `Max` incidently.

Reply:
Is what you mean to use `Compare` method?

Reply:
```
var first = arguments.At(0);

return input.ToNumberValue() < first.ToNumberValue()
  ? first
  : input
  ;
```